trashcli/list_mount_points.py

```diff
@@ -2,7 +2,7 @@
 
 def mount_points():
     try:
-        return list(mount_points_from_getmnt())
+        return list(s.decode('utf-8') for s in mount_points_from_getmnt())
```

Comment:
IMHO this won't work since `mount_points_from_getmnt()` returns instances of `Filesystem`.
You'll have to update the code in [line 74](https://github.com/andreafrancia/trash-cli/pull/115/files#diff-9b547c67d5ee9b0451c30680f46ad81cR74):
```python
yield Filesystem(entry.contents.mnt_dir.decode('utf-8'),
                 entry.contents.mnt_type.decode('utf-8'),
                 entry.contents.mnt_fsname.decode('utf-8'))
```